lib/ContainerManager.js

```diff
@@ -408,6 +435,76 @@ Server.prototype.routes.get['containers'] = function(req, res, next) {
   });
 };
 
+/**
+ * Delete the docker container from the server. THIS OPERATION CANNOT BE REVERSED!
+ * By default, will not remove running containers.
+ * Call with ?force=true delete running containers too.
+ */
+Server.prototype.routes.del['containers/:id'] = function(req, res, next) {
+  var self = this
+
+  // delete/remove the container and update the build entry in the DB
+  var container = new Container({
+    docker: this.config.docker,
+    containerId: req.params.id,
+    log: this.log,
+  })
+
+  log_container_events(container)
+
+  function handle_error(err, msg){
```

Comment:
It seems like this could probably be a reusable error handler on the `Server` prototype itself.